docs/guides/lint-arazzo.md

```diff
@@ -0,0 +1,73 @@
+# Lint Arazzo with Redocly CLI
+
+Arazzo is an open standard for describing multi-step API calls in sequence.
+While under development, this standard was also known as "OpenAPI Workflows".
+Redocly CLI offers support for checking that your Arazzo files are valid.
+
+{% admonition type="info" name="Experimental Arazzo support" %}
+This feature is at an early stage, please send us lots of feedback!
+{% /admonition %}
+
+## Lint an Arazzo file
+
+Use your existing Arazzo files, or use the examples in the [Museum API project](https://github.com/Redocly/museum-openapi-example) if you'd prefer to use sample data to try things out.
+
+**Pro-tip:** linting is much more interesting if the file actually does contain problems.
+Be your own chaos monkey and introduce some errors before you proceed!
+
+Lint using a command like the following:
+
+```bash
+redocly lint arazzo/museum-api.arazzo.yaml
+```
+
+If the file does not match the specification, the tool shows the details of each error that it finds.
+
+{% admonition type="info" name="Validation only" %}
+No additional rules or configuration are available for Arazzo in the current version of Redocly CLI; the tool merely checks that the file meets the specification.
+{% /admonition %}
+
+## Choose output format
+
+Since Redocly CLI is already a fully-featured lint tool, additional features such as a choice of formats are already included.
```

Comment:
```suggestion
Since Redocly CLI is already a [fully-featured lint tool](../cli/command/lint.md), additional features such as a choice of formats are already included.
```